File: tests/claude-plugin-integration/test-repos/fsharp-sample/PaymentService.fs
module PaymentService =
    open User

    let processPayment (user: User.User) amount =
        printfn "Processing $%.2f for %s" amount user.Name
        true

    let refundPayment (user: User.User) amount =
        printfn "Refunding $%.2f to %s" amount user.Name
        true

File: tests/claude-plugin-integration/test-repos/java-sample/PaymentService.java
public class PaymentService {
    public boolean processPayment(User user, double amount) {
        System.out.printf("Processing $%.2f for %s%n", amount, user.getName());
        return true;
    }

    public boolean refundPayment(User user, double amount) {
        System.out.printf("Refunding $%.2f to %s%n", amount, user.getName());
        return true;
    }
}
